### PR TITLE
PR cleanup

### PR DESCRIPTION
**Pull request by mapcat**

### CODE REVIEW

**Comment by rickmastfan67** [comment]
mapcat, just so you know, every time you add another change, a new email is sent out.  Got over 61 emails from this PR today alone.  I would recommend that if you need to do a mass renaming project, do it before you create the PR so we aren't all email spammed on every file rename.  Thanks man. :)

**Comment by mapcat** [comment]
Sorry!!

But why would it be sending out emails?? I never get emails when anyone else renames anything. I don't think I ever get emails except when someone responds to one of my pulls, issues, etc.

Tell me what I need to change in the settings to fix this. And also how I would do a "mass renaming" without editing the files one at a time...I'm not aware of any other way to do that. Not that I will need to do it again, since I'm done with these...

Again, sorry.

**Comment by rickmastfan67** [comment]
Pretty much, all you need to do is not create the PR till you're ready.  Simple as that.  As long as all the renames/additions to each file are done on your branch or your master before the PR is created, no mass email spam. :)  Since we're all 'contributors' to the TravelMapping, we automatically get all emails when a new PR is pushed, as well as any posts or new commits that are added to it.

**Comment by michihdeu** [comment]
I've never got any email from GitHub...

**Comment by ovoss** [comment]
I second rickmastfan67's request. Not combining all your changes into a single PR (or several large PRs, if you're doing it in phases) not only makes it hard to follow what you're doing and any responses, but makes it hard for me to pick out non-GitHub emails in the clutter I need to deal with. 


When I did my CA cleanup (just Interstates, US routes, and their business routes), I split it into several phases, usually just one PR per day. That involved fewer files, but more work for each file (such as a painful conversion of most waypoint labels on US 50 and US 101 to exit numbers).

**Comment by mapcat** [comment]
Thanks again for pointing this out. I still don't see why it would be happening to you. Like michih, I've never received an email from GitHub unless it specifically related to something I had submitted. This seems to be something you can personalize in settings. 

When I created the pull request, I thought I was going to be finished for the day. But I ended up having more time and wanted to

**Comment by mapcat** [comment]
Hard to do this on a mobile. 

Anyway, I accidentally closed the pull request and then decided I should probably leave it closed until I'm finished with the cleanup, since I'll have done more time tonight. Im guessing that will prevent it from sending out more emails?

And to finish the thought from the previous comment, I thought it would be better to collect all the changes into one pull request rather than opening several over a few days.  It's not possible to NOT add a file change to an open pull request, is it?

Again, my apologies.

**Comment by michihdeu** [comment]
You could commit and sync your changes without a pull request. The pull request is usally the last step.

**Comment by si404** [comment]
I've now got a second thread outside the 100-email thread (thankfully gmail threaded them!) The close-pull-open was three emails.

If you have a pull request open, then committing and syncing (if using the web interface, all commits are synced) to your branch adds them to that pull request (see my open pull request that I made sure to post before midday EDT Friday to have it pulled in, but alas, and then I added corrections as I reviewed Northern Ireland).

I'm going to pull this one in, and you can commit away on your branch to your hearts content without emailing those who have chosen to subscribe to this repository until your next pull request.

**Comment by mapcat** [comment]
Thank you!
